src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -116,7 +118,7 @@ const GetTradeQuotes = () => {
   return <></>
 }
 
-export const TradeInput = memo(({ isCompact }: TradeInputProps) => {
+export const TradeInput = ({ isCompact }: TradeInputProps) => {
```

Comment:
This fixed this issue:

https://discord.com/channels/554694662431178782/1242670834930290688/1245874260518375585